template-parts/homepage-landing.php

```diff
@@ -10,13 +10,13 @@
     <div class="swiper-wrapper">
         <!-- Slides -->
         <div class="swiper-slide">
-        	<div class="content">Since 2005, <span style="color:#CE2727">8,730 deaths</span> have been reported in Texas custody.</div>
+        	<div class="content">Since 2005, <span style="color:#CE2727"><span id="cdrTotal"></span> deaths</span> have been reported in Texas custody.</div>
```

Comment:
I know this isn't for this PR, but is there a reason the colors are inline? could we accomplish this with a class and some css. Also could we continue to stick to BEM for our class names.

Reply:
It looks like the dates could be made dynamic as well. These are in the dataset meta as well.

Reply:
Added a color class: text--color-red to the sass and used that instead. Also made the dates dynamic.